app/Notifications/UpdatePlaylistNotification.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class UpdatePlaylistNotification extends Notification
+{
+
+    use Queueable;
+
+    protected $pageUrl;
+
+    public $sender;
+    public $project;
+    public $playlist;
+    public $token;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param $sender
+     * @param $project
+     * @param $playlist
+     * @param $token
+     */
+    public function __construct($sender, $project, $playlist, $token)
+    {
+        $this->sender = $sender;
+        $this->project = $project;
+        $this->playlist = $playlist;
+        $this->token = $token;
```

Comment:
Seeing this token variable in controllers and event classes, is it being in use any-where in use? I don't see it's usage anywhere.